features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioGattServer.cpp

```diff
@@ -709,20 +701,15 @@ ble_error_t GattServer::areUpdatesEnabled(
 ) {
     for (size_t idx = 0; idx < cccd_cnt; idx++) {
         if (characteristic.getValueHandle() == cccd_handles[idx]) {
-            uint16_t conn_id = 0;
-            uint16_t conn_found = 0;
-
-            while ((conn_found < DM_CONN_MAX) && (conn_id < CONNECTION_ID_LIMIT)) {
+            for (dmConnId_t conn_id = DM_CONN_MAX; conn_id > DM_CONN_MAX; --conn_id) {
```

Comment:
Shouldn't `DM_CONN_ID_NONE` be used in comparison here?

Reply:
yes, thanks for the fix @paul-szczepanek-arm